doc/src/driver/compile.rst

```diff
@@ -28,6 +28,10 @@ Enable implicit offload copying
 
 Disable fast math optimization
 
+.. option:: --tune
+
+Enable tuning for the GPU kernels
```

Comment:
This should say "Perform an exhaustive search to find the fastest version.", this flag doesnt need to apply to just the GPU kernels.

Reply:
Done.